### PR TITLE
Emi.new docs merged

### PR DESCRIPTION
**Pull request by tenmaster**
This PR includes:
- 04 UserProfile
- 05 LinkingAccounts
- 10 Customizing Lock

All these are new docs, that were pending due to the requirement of an lock v2 update.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by volbap** [comment]
+1 @mpaktiti

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/android/04-user-profile.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+This tutorial will show you how to use Lock10 to get the user's profile data in your android apps with Auth0.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+:::
+
+ <%= include('../../_includes/_github', {
```
Comment (mpaktiti) [inline articles/native-platforms/android/04-user-profile.md]: @tenmaster Hi, this file does not exist, please see [this PR](https://github.com/auth0/docs/pull/2119/files) and update the docs in this one accordingly. Thank you.

^ @chenkie

**Pull request merged**